server/src/main/java/com/thoughtworks/go/server/service/PipelineScheduler.java

```diff
@@ -86,6 +86,9 @@ public void initialize() {
             public void onEntityConfigChange(PipelineConfig pipelineConfig) {
                 synchronized (pipelines) {
                     addPipelineIfNotPresent(pipelineConfig, pipelines);
```

Comment:
NOTE: doesn't need to handle pipeline update scenario as the scheduler adds only the 'pipeline_name' to the `scheduleCheckQueue`.